api_tests/tests/lightning_routes.ts

```diff
@@ -18,18 +19,29 @@ describe("Lightning routes tests", () => {
             const bodies = (await SwapFactory.newSwap(alice, bob, true))
                 .hanEthereumEtherHalightLightningBitcoin;
 
-            await expect(
-                alice.cnd.createHanEthereumEtherHalightLightningBitcoin(
-                    bodies.alice
-                )
-            ).rejects.toThrow("lightning is not configured.");
-            try {
-                await bob.cnd.createHanEthereumEtherHalightLightningBitcoin(
-                    bodies.bob
-                );
-            } catch (err) {
-                expect(err.status).toBe(400);
-            }
+            const aliceResponse = alice.cnd.createHanEthereumEtherHalightLightningBitcoin(
+                bodies.alice
+            );
+            const bobResponse = bob.cnd.createHanEthereumEtherHalightLightningBitcoin(
+                bodies.bob
+            );
+
+            await expect(aliceResponse).rejects.toMatchObject(
```

Comment:
📣 This only looks at the properties of the object. You need to use `toStrictEqual` instead.

Reply:
Good call, thank you :)

Reply:
That doesn't seem to work though?

![image](https://user-images.githubusercontent.com/5486389/80781106-803bd380-8bb4-11ea-85cd-339e4ae4f3f2.png)

Reply:
It should fail because of wrong status AND wrong detail. It seems to compare just the message of the error ...

Reply:
Not instantiating the problem seems to do the trick combined with `toMatchObject`:

![image](https://user-images.githubusercontent.com/5486389/80781664-98145700-8bb6-11ea-88d3-1db68411f69a.png)

Reply:
Ok, weird, the jest doc is basically misleading.

Reply:
Ok I read the doc again and now I understand, all good :)